specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/cadl/cadl-project.yaml

```diff
@@ -0,0 +1,17 @@
+emitters:
```

Comment:
Can we please restructure the cadl projects according to the guidelines at https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/cadl-structure-guidelines.md.

Reply:
Based on that guidelines, @weshaggard would this be the correct structure based on the pre-existing contents of the cognitiveservices directory?

Example:
````
- specification/cognitiveservices/data-plane/AnomalyDetector/
  - cadl files
  - stable/ (pre-existing stable swagger)
  - preview/ (pre-existing preview swaggers)
 ```

Reply:
Alternately, to follow the guideline more closely it could be that we move everything under the current AnomalyDetector/ directory to a data-plane folder, like the following:
```
- specification/cognitiveservices/data-plane/AnomalyDetector/
  - data-plane (move all current swagger related content to this folder)
  - add cadl files at this level
```

Reply:
I would expect this to be more like:
```
- specification/cognitiveservices
  - AnomoalyDetector  (cadl project and files go under here. 
  - data-plane
  - resource-manager
```

> Would this move affect any existing pipelines? Or tooling?

Those tools are actively being worked on /cc @raych1